DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/Enrol/EnrolDelegateSummary.cshtml

```diff
@@ -0,0 +1,150 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.Enrol
+@model EnrolSummaryViewModel
+@{
+    ViewData["Title"] = "EnrolDelegateSummary";
+    var routeParamsForCancelLink = new Dictionary<string, string> {
+    { "delegateId", Model.DelegateId.ToString() },
+  };
+    var prefillDay = Model.CompleteByDate?.Day.ToString() ?? "";
+    var prefillMonth = Model.CompleteByDate?.Month.ToString() ?? "";
+    var prefillYear = Model.CompleteByDate?.Year.ToString() ?? "";
+}
+<div class="nhsuk-grid-row">
+    <div class="nhsuk-grid-column-full">
+        <form method="post">
+            <div class="nhsuk-u-margin-bottom-4">
+                <h2 class="nhsuk-heading-xl">Enrolment details</h2>
+            </div>
+            <dl class="nhsuk-summary-list">
+                <div class="nhsuk-summary-list__row">
+                <dt class="nhsuk-summary-list__key">
+                    Delegate
+                </dt>
+                <dd class="nhsuk-summary-list__value">
+                        @Model.DelegateName
+                </dd>
+                <dd></dd>
+    </div>
+    <div class="nhsuk-summary-list__row">
+        <dt class="nhsuk-summary-list__key">
+            Activity
+        </dt>
+        <dd class="nhsuk-summary-list__value">
+                        @Model.ActivityName
+        </dd>
+
+        <dd class="nhsuk-summary-list__actions">
+            <a asp-action="Index" asp-route-delegateId="@Model.DelegateId" asp-route-delegateName="@Model.DelegateName">
+                Change
+            </a>
+        </dd>
+    </div>
+    <div class="nhsuk-summary-list__row">
+        <dt class="nhsuk-summary-list__key">
+            Complete by date
+        </dt>
+        <dd class="nhsuk-summary-list__value">
+                        @if (Model.CompleteByDate.HasValue)
+                        {
+                            @Model.CompleteByDate
```

Comment:
This is displaying with a 00:00:00 timestamp - can we display as short date instead? dd/MM/yyyyy